### PR TITLE
Add health check on docker compose

### PR DESCRIPTION
**Pull request by antoalconti**
#### Board:
* [Ticket #NUMBER_OF_THE_TICKET](https://www.notion.so/rootstrap/One-off-Tasks-e7bed551a599475ea88f9c825c9cfdb7?p=866941cff01349f3af35d61f93e315e8&pm=s)
---
#### Description:

--- Checks for web and db were added, now on status we can chech if healthy or not
#### Notes:
*
---
#### Tasks:
- [x] Add each element in this format
---
#### Risk:
*
---
#### Preview:
image

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -18,6 +18,12 @@ services:
       - db
     links:
       - db:db
+    healthcheck:
+        test: ["CMD", "curl", "-f", "http://localhost:3000"]
+        interval: 1m30s
+        timeout: 10s
+        retries: 3
+        start_period: 40s
```
Comment (JulianPasquale) [inline docker-compose.yml:26]: I feel these values are a bit high 🤔 does it really take 40s for the server to start?

Comment (antoalconti) [reply docker-compose.yml:26]: i use default values, not sure which are the average, maybe 20 seconds? @JulianPasquale

**Review by axelpontetto — APPROVED** [review]

**Review by santib — APPROVED** [review]

File: docker-compose.yml (hunk 1 above)
Comment (santib) [reply docker-compose.yml:26]: where did you take the defaults from?

Comment (antoalconti) [reply docker-compose.yml:26]: https://docs.docker.com/reference/compose-file/services/#healthcheck

**Pull request merged**